redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -163,13 +163,13 @@ object Output {
 
   case object StringOutput extends Output[String] {
     protected def tryDecode(text: String): String =
-      if (text.startsWith("+"))
+      if (text.startsWith("$"))
```

Comment:
I think that what @mijicd suggested here is keep the current state from `master` and use `MultiStringOutput` where you need Bulk string.

Reply:
good catch, fixing it